config/obsoletion.yml

```diff
@@ -6,6 +6,13 @@
 
 # Cop parameters that have been changed
 # Can be treated as a warning instead of a failure with `severity: warning`
+changed_enforced_styles:
+  - cops: RSpec/VerifiedDoubleReference
+    parameters: EnforcedStyle
+    value: string
+    reason: String references are not verifying unless the class is loaded.
+    severity: warning
+
```

Comment:
Instead of these lines, I’d add this to `changed_parameters`:
```yaml
  - cops: RSpec/VerifiedDoubleReference
    parameters: EnforcedStyle
    reason: String references are not verifying unless the class is loaded.
```